launching-software/security.md

```diff
@@ -82,7 +82,7 @@ Static analysis is an important part of the development process, and is required
 - [**Dependency analysis**](#dependency-analysis), where the Ruby gems, Python modules, and JavaScript packages your app uses are checked against a list of known vulnerabilities.
 - [**Code security analysis**](#code-analysis), in which your code is checked against a list of antipatterns.
 
-There are tools for JS, Ruby, and Python, and you are encouraged to set up this scanning early on in the development cycle to prevent unexpected delays when it's time to get your ATO. Note that there are _many_ tools out there for doing code style linting - see the [18F Development Guide](https://github.com/18F/development-guide) for recommendations. This page is specifically security-focused.
+There are tools for JS, Ruby, and Python, and you are encouraged to set up this scanning early on in the development cycle to prevent unexpected delays when it's time to get your ATO. Note that there are _many_ tools out there for doing code style linting - see the [Development Guide](https://github.com/18F/development-guide) for recommendations. This page is specifically security-focused.
```

Comment:
```suggestion
There are tools for JS, Ruby, and Python, and you are encouraged to set up this scanning early on in the development cycle to prevent unexpected delays when it's time to get your ATO. Note that there are _many_ tools out there for doing code style linting - see the [Development Guide](https://engineering.18f.gov/language-selection/) for recommendations. This page is specifically security-focused.
```